cypress/integration/dryRunner/pirateMechanics.js

```diff
@@ -0,0 +1,80 @@
+import s from './selectors'
+
+const DRAW_AND_DISCARD_DECK_ID =
+  'NU4xLDVOMiw1TjMsNU4yMyw1TjQsNU42LDVONjIsNU42Myw1TjY3LDVOMTIsNU4xNCw1TjIy'
+const SNAKE_EYES_DECK_ID =
+  'NU4xLDVOMiw1TjMsNU4yMyw1TjQsNU41LDVONiw1TjYyLDVONjMsNU42Nyw1TjY2LDVOMzM'
+
+describe('Dry-runner — Discard and Draw', () => {
```

Comment:
Shall we have a file per pirate card or is one enough?

Reply:
For now I think this is good enough, we'll change to draw + discard if we add more tests